nemo-physical/src/management/database.rs

```diff
@@ -768,6 +769,31 @@ impl DatabaseInstance {
         }
     }
 
+    /// Evaluates a [`ExecutionTree`] until until it finds the first row in the result and returns it.
+    /// Returns `None` if the tree evaluates to the empty table.
+    ///
+    /// # Panics
+    /// Panics if the tables that are being loaded by the [`ExecutionTree`] are not available in memory.
+    /// Also panics if the [`ExecutionTree`] wants to perform a project/reorder operation on a non-materialized trie.
+    fn evaluate_execution_tree_until_first(
```

Comment:
We have `until_first` here and `first_match` down below, since they mean the same thing, they should have a uniform name (`until_first_match`, e.g.).